lib/schema.js

```diff
@@ -343,3 +343,5 @@ export const toSchema = (browser, options = {}) => {
 
   return schema;
 };
+
+export const getAllRegisteredSchemaUris = () => Object.keys(schemaRegistry);
```

Comment:
Can you move this next to the other schema registry functions (`registerSchema`, `unregisterSchema`). I think that will make maintenance easier.